ldk/javascript/src/whisper/react/component-types.ts

```diff
@@ -0,0 +1,81 @@
+import { ReactNode } from 'react';
+import * as Whisper from '../types';
+import { WhisperHandler } from '../types';
+
+/**
+ * @internal
```

Comment:
Will these annotations prevent these from being picked up in our generated documentation?